src/main/java/bio/terra/buffer/service/resource/FlightManager.java

```
@@ -40,13 +40,30 @@ public FlightManager(
 
   /** Submit Stairway Flight to create resource. */
   public Optional<String> submitCreationFlight(Pool pool) {
-    return transactionTemplate.execute(status -> createResourceEntityAndSubmitFlight(pool, status));
+    try {
+      return transactionTemplate.execute(
+          status -> createResourceEntityAndSubmitFlight(pool, status));
+    } catch (RuntimeException e) {
```

Is it important to catch all exceptions here and return an empty value? At the very least, if we're catching all RuntimeExceptions we should log out the exception message before returning, so the stack traces show up in our logs.

---

This method is one part of the scheduler service scheduling flight. We don't want scheduler to continue even one flight submit failed.
Added logs

---

Please add a comment to that effect.

---

removed